js/answer.js

```diff
@@ -0,0 +1,18 @@
+/* Aufgabe: Antwort ist hidden. Mit Klick auf Show Answer, die Antwort anzeigen
+erneuter Klick auf Hide Answer, die Antwort verstecken */
+
+let button = document.querySelector("[data-js=button]");
+let answer = document.querySelector("[data-js=answer]");
```

Comment:
You do not reassign a new value to this variable later. Therefore, I would recommend you to use `const` instead of `let`.